addons/mail/static/src/components/attachment_image/attachment_image.xml

```diff
@@ -20,15 +20,17 @@
                             <i class="fa fa-spin fa-spinner"/>
                         </div>
                     </t>
-                    <t t-if="attachmentImage.attachment.isDeletable">
-                        <div class="o_AttachmentImage_imageOverlay position-absolute top-0 bottom-0 start-0 end-0 text-right p-2 text-white opacity-0 opacity-100-hover">
-                            <div class="o_AttachmentImage_action o_AttachmentImage_actionUnlink btn btn-sm btn-dark rounded opacity-75 opacity-100-hover"
+                    <div class="o_AttachmentImage_imageOverlay position-absolute top-0 bottom-0 start-0 end-0 p-2 text-white opacity-0 opacity-100-hover d-flex align-items-end  flax-wrap flex-column">
```

Comment:
single whitespace between classnames (there are 2 between `align-items-end` and `flax-wrap`)